src/_pytest/pytester.py

```diff
@@ -1267,9 +1268,9 @@ def runpytest_subprocess(self, *args, timeout=None) -> RunResult:
         return self.run(*args, timeout=timeout)
 
     def spawn_pytest(
-        self, string: str, expect_timeout: float = 10.0
+        self, string: str, expect_timeout: float = 10.0, args: List[str] = None
```

Comment:
```suggestion
        self, string: str, expect_timeout: float = 10.0, args: Optional[Sequence[str]] = None
```

Alternatively, accepts `*args: str`. That would be more consistent with `runpytest` for example, and can replace `string`.